plugins/SegmentEditor/javascripts/Segmentation.js

```diff
@@ -163,10 +163,13 @@ Segmentation = (function($) {
                 + ' ' + self.translations['General_DefaultAppended']
                 + '</span>';
             var comparisonService = window.CoreHome.ComparisonsStoreInstance;
-            if (comparisonService.isComparisonEnabled()
-                || comparisonService.isComparisonEnabled() === null // may not be initialized since this code is outside of Vue
+            if (
+                comparisonService.isComparisonEnabled() ||
+                comparisonService.isComparisonEnabled() === null // may not be initialized since this code is outside of Vue
             ) {
-                listHtml += '<span class="compareSegment allVisitsCompareSegment" title="' + _pk_translate('SegmentEditor_CompareThisSegment') + '"></span>';
+                const className = 'compareSegment allVisitsCompareSegment ' + (self.segmentAccess === 'write' ? 'allVisitsCompareSegment--write' : '');
+                const title = _pk_translate('SegmentEditor_CompareThisSegment');
+                listHtml += '<span class="' + className + '" title="' + title + '"></span>';
```

Comment:
ℹ️ Make sure the compare button is always align

| anonymous | connected |
|---|---|
| <img width="309" height="242" alt="Screenshot 2025-11-24 at 15 37 31" src="https://github.com/user-attachments/assets/df2869fa-c968-4426-aa66-b3664e0bd6fe" /> | <img width="354" height="183" alt="Screenshot 2025-11-24 at 15 34 21" src="https://github.com/user-attachments/assets/45986365-2d32-4cc3-ba84-5a164b05a9f5" /> |